### DISCUSSION TITLE
Caching api 

### DISCUSSION THREAD

**Discussion by Sangdt** [vercel/next.js#10887] [Help]
So how to do caching on api routes with nextjs. Currently i'm using apicache for my api routes but i dont think it's a good pratices. Is there anyway or documents how to cache the return of the api routes ?

**Comment by mousedown** [comment]
How about [etag](https://www.npmjs.com/package/etag)

**Comment by dohomi** [comment]
@Sangdt if you use Vercel/Zeit you can declare a `Cache-Control` header response
